wrappers/ios/libindy-pod/Indy/Wrapper/IndyDid.h

```diff
@@ -93,6 +94,10 @@
  @endcode
  
  @param identityJSON Identity information as json. See example above.
+    {
+       "did": string, (required)
+       "verkey": string (optional, can be avoided if did is cryptonym: did == verkey),
```

Comment:
Is this saying a string is not needed if this is being first the first time the DID is being stored, but if a DID is being updated, then the verkey would be needed? Additionally, Is the verkey required to be in the wallet already to perform this update?

Reply:
Also, do we need to do a check on the format of the key (e.g. base58btc)

Reply:
`Is this saying a string is not needed if this is being first the first time the DID is being stored, but if a DID is being updated, then the verkey would be needed?` - make sense. Thanks

`Additionally, Is the verkey required to be in the wallet already to perform this update?` - verkey is already stored together with DID in the wallet. 
verkey is optional only if cryptonym DID was passed. In this case Verkey will be the same as DID.
If DID is short and verkey isn't passed the validation error will be thrown.

Reply:
Actualy, there is no any changes in the function definition. I just noticed that `identityJSON` definition was missed in some wrappers.